src/test/java/seedu/partyplanet/logic/commands/CommandTestUtil.java

```diff
@@ -97,6 +106,22 @@
                 .withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND).build();
     }
 
+    public static final EEditCommand.EditEventDescriptor DESC_CNY;
+    public static final EEditCommand.EditEventDescriptor DESC_EASTER;
+
+    static {
+        DESC_CNY = new EditEventDescriptorBuilder()
+                .withName(VALID_NAME_CNY)
+                .withDate(VALID_DATE_CNY)
+                .withRemark(VALID_REMARK_CNY)
+                .build();
+        DESC_EASTER = new EditEventDescriptorBuilder()
+                .withName(VALID_NAME_EASTER)
+                .withDate(VALID_DATE_EASTER)
+                .withRemark(VALID_REMARK_EASTER)
+                .build();
+    }
```

Comment:
The static block here seems unnecessary. I get that is following the format of the lines above. Perhaps can refactor that too


```suggestion
    public static final EEditCommand.EditEventDescriptor DESC_CNY = new EditEventDescriptorBuilder()
            .withName(VALID_NAME_CNY)
            .withDate(VALID_DATE_CNY)
            .withRemark(VALID_REMARK_CNY)
            .build();
    public static final EEditCommand.EditEventDescriptor DESC_EASTER = new EditEventDescriptorBuilder()
            .withName(VALID_NAME_EASTER)
            .withDate(VALID_DATE_EASTER)
            .withRemark(VALID_REMARK_EASTER)
            .build();
```

Reply:
Keeping the original because it looks a bit neater